### PR TITLE
Module folder

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 /config-local
 # Maven
 **/target/
+# Travis tests
```
Comment (henning-gerhardt) [inline .gitignore]: This change is independent of Travis - it is more a general test usage.

**Comment by henning-gerhardt** [comment]
Your latest changes break a `mvn clean install` or `mvn clean install -Pall-tests` command call.

```
[INFO] --- maven-resources-plugin:3.0.2:copy-resources (copy-files-on-build) @ kitodo-file-management ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Kitodo Production .................................. SUCCESS [  0.574 s]
[INFO] OpacPica-Plugin .................................... SUCCESS [  2.881 s]
[INFO] Kitodo - Data Management ........................... SUCCESS [  4.005 s]
[INFO] Kitodo - API ....................................... SUCCESS [  0.370 s]
[INFO] Kitodo - File Management ........................... FAILURE [  2.216 s]
[INFO] Kitodo - Persistent Identifier ..................... SKIPPED
[INFO] Kitodo - Docket .................................... SKIPPED
[INFO] Kitodo - Command ................................... SKIPPED
[INFO] Kitodo - Core ...................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 10.329 s
[INFO] Finished at: 2017-08-23T08:59:11+02:00
[INFO] Final Memory: 44M/430M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-resources-plugin:3.0.2:copy-resources (copy-files-on-build) on project kitodo-file-management: The parameters 'outputDirectory' for goal org.apache.maven.plugins:maven-resources-plugin:3.0.2:copy-resources are missing or invalid -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginParameterException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :kitodo-file-management

```

**Comment by Beacze** [comment]
It is necessary to add modules folder tothe local directory.

**Comment by henning-gerhardt** [comment]
As I added `moduleFolder` property to my local kitodo_config.property file mvn calls are working now. But the `Kitodo Command` module is still installed to plugin directory. This must be fixed.

**Comment by Beacze** [comment]
Command module build changed.

**Pull request merged**